recipes/abess/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "abess" %}
+{% set version = "0.4.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4b3d615efad34f0a24255594b27cec9ff2aae41d2a70812308c4bcceb02b45bb
+
+build:
+  number: 0
+  skip: True  # [py<35 and win]
```

```suggestion
  skip: True  # [win]
```

We only run 37 and above anyway